src/test/scala/com/codacy/api/FileHelperTest.scala

```diff
@@ -0,0 +1,43 @@
+package com.codacy.api
+
+import com.codacy.api.helpers.FileHelper
+import org.scalatest.{FlatSpec, Matchers}
+
+class FileHelperTest extends FlatSpec with Matchers {
+
+  "FileHelper" should "withTokenAndCommit" in {
+
+    val result: Either[String, Boolean] = FileHelper.withTokenAndCommit(projectToken = Some("PROJECT-TOKEN-DUMMY-VALUE")) {
+      case (projectToken, commitUUID) =>
+
+        projectToken shouldNot be('empty)
+        commitUUID shouldNot be('empty)
+
+        Right(true)
+    }
+
+    result should be('right)
+
```

Comment:
I think we should test this cases with (import org.scalatest.EitherValues._):

either.right.value should be (true)